packages/react/src/components/DataTable/DataTable-story.js

```diff
@@ -356,3 +368,5 @@ export const WithCheckmarkColumns = () => {
     </DataTable>
   );
 };
+
+export const Playground = () => require('./stories/default').default(props());
```

Comment:
We should try and inline the example here with knobs so that folks can see it in the story source 👀  Otherwise they just see the `require` statement

Reply:
the default story is unused then, so I think we can just remove it